packages/triceratops/PKGBUILD

```diff
@@ -0,0 +1,68 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+# Contributor: speps <speps at aur dot archlinux dot org>
+
+_plugin_uri='http://nickbailey.co.nr/triceratops'
+pkgname=triceratops
+pkgver=0.5.0
+pkgrel=1
+pkgdesc='Polyphonic synthesizer LV2 plugin'
+arch=(aarch64 x86_64)
+url='https://sourceforge.net/projects/triceratops/'
+license=(GPL3)
+groups=(lv2-plugins pro-audio)
+depends=(gtkmm)
+makedepends=(python lv2)
+checkdepends=(lv2lint)
+optdepends=('lv2-host: for loading the LV2 plugin')
+provides=($pkgname.lv2)
+conflicts=($pkgname.lv2)
+source=("https://downloads.sourceforge.net/project/triceratops/${pkgname}_v${pkgver}_Oct_2022.tar.gz"
+        'triceratops-gui-instantiate.patch'
+        'triceratops-remove-printf.patch'
+        'triceratops-symbol-visibility.patch'
+        'triceratops-ttl.patch')
```

Comment:
If these patches are identical to the PRs, you can also source them directly using `triceratops-gui-instantiate.patch::https://patch-diff.githubusercontent.com/raw/thunderox/triceratops/pull/11.patch`

Reply:
Unfortunately, since I created these the PRs with these patches independently, they contain conflicting whitespace changes and can not be used as is.